services/rest/websocket/ack.go

```diff
@@ -0,0 +1,38 @@
+package websocket
+
+import (
+	"time"
+
+	"github.com/odpf/raccoon/metrics"
+	"github.com/odpf/raccoon/serialization"
+	"github.com/odpf/raccoon/services/rest/websocket/connection"
+)
+
+var AckChan = make(chan AckInfo)
+
+type AckInfo struct {
+	MessageType     int
+	RequestGuid     string
+	Err             error
+	Conn            connection.Conn
+	serializer      serialization.SerializeFunc
+	TimeConsumed    time.Time
+	AckTimeConsumed time.Time
+}
+
+func AckHandler(ch <-chan AckInfo) {
```

Comment:
Same as above. Can it be just `Handler`?

Reply:
We cannot, It's not under the `ack` package.